### ISSUE TITLE
Internal reference to an equation does not work in docx

### ISSUE THREAD

**Issue by tkarna** [timtylin/scholdoc#5]
A minimal example:

```
~~~math #firstequation
\frac{\partial C}{\partial t} = \nabla \cdot \left( \nu \nabla C \right ),
~~~

See equation (#firstequation).
```

converted to docx with

```
scholdoc test_minimal.md -o test_minimal.docx
```

does not produce the equation number nor the internal reference to equation (1) in the docx file. Converting to tex works fine.

**Comment by timtylin** [comment]
> I seems like that would be sufficient for many forms of academic writing and render scholdoc usable for such tasks, while the generic implementation is being developed.

Yes, I've been planning to do this for a while now as part of an overhaul of the DOCX writer, but I just don't have the time to do it yet, and I've been busy trying to graduate. I'm not all that familiar with OOML as well, but I have collected some resources and help on this matter. I'll just need a day or two of uninterrupted time to get into the headspace of untangling OOML, and then I'll tackle both math and float output support when I'm there.
